src/code/game.c

```diff
@@ -442,7 +436,7 @@ void GameState_Init(GameState* gameState, GameStateFunc init, GraphicsContext* g
     VisCvg_Init(&sVisCvg);
     VisZBuf_Init(&sVisZBuf);
     VisMono_Init(&sVisMono);
-    if (VI_MODE_EDITOR_INACTIVE) {
+    if ((R_VI_MODE_EDIT_STATE == VI_MODE_EDIT_STATE_INACTIVE) || !OOT_DEBUG) {
```

Comment:
personally would prefer `OOT_DEBUG` checks to come first (here and below)
```suggestion
    if (!OOT_DEBUG || (R_VI_MODE_EDIT_STATE == VI_MODE_EDIT_STATE_INACTIVE)) {
```

Reply:
I agree, but putting `!OOT_DEBUG` first did not match debug :/

Reply:
oh it matches in `GameState_Init` but not `GameState_Destroy` :pensive: 

it would match in `GameState_Destroy` with just one more use of `gameState` for the compiler to put it in s0...
anyway whatever for this PR